html/includes/graphs/generic_multi_seperated.inc.php

```diff
@@ -1,19 +1,41 @@
 <?php
+/*
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *
+ * @package    LibreNMS
+ * @subpackage graphs
+ * @link       http://librenms.org
+ * @copyright  2017 LibreNMS
+ * @author     LibreNMS Contributors
+*/
+
+use LibreNMS\Config;
 
 require 'includes/graphs/common.inc.php';
 
+if (Config::get('graph.stacked') == true) {
+    $transparency = 95;
```

Comment:
What's the reason for the different transparency values in the files?

Overall we should put this block into a function to make it re-usable

Reply:
this graph is used on device ports overview and lower values make it unpleasant to view (too much transparency).

Reply:
Is that not the same for other pages then?